packaging/installer/UPDATE.md

```diff
@@ -41,17 +45,23 @@ bash <(curl -Ss https://my-netdata.io/kickstart-static64.sh) --no-updates
 
 ### Auto-update
 
-_Please, consider the risks of running an auto-update. Something can always go wrong. Keep an eye on your installation, and run a manual update if something ever fails._
+_Please, consider the risks of running an auto-update. Something can always go wrong. Keep an eye on your installation,
+and run a manual update if something ever fails._
 
-Calling the `netdata-installer.sh` with the `--auto-update` or `-u` option will create the `netdata-updater` script under 
-either  `/etc/cron.daily/`, or `/etc/periodic/daily/`. Whenever the `netdata-updater` is executed, it checks if a newer nightly build is available and then handles the download, installation and Netdata restart.  
+Calling the `netdata-installer.sh` with the `--auto-update` or `-u` option will create the `netdata-updater` script
+under either  `/etc/cron.daily/`, or `/etc/periodic/daily/`. Whenever the `netdata-updater` is executed, it checks if a
+newer nightly build is available and then handles the download, installation and Netdata restart.  
 
-Note that after Jan 2019, the `kickstart.sh` one-liner `bash <(curl -Ss https://my-netdata.io/kickstart.sh)` calls the `netdata-installer.sh` with the auto-update option. So if you just run the one-liner without options once, your Netdata will be kept auto-updated.
+Note that after Jan 2019, the `kickstart.sh` one-liner `bash <(curl -Ss https://my-netdata.io/kickstart.sh)` calls the
+`netdata-installer.sh` with the auto-update option. So if you just run the one-liner without options once, your Netdata
+will be kept auto-updated.
 
 ## You downloaded a binary package
 
-If you installed it from a binary package, the best way is to **obtain a newer copy** from the source you got it in the first place.
+If you installed it from a binary package, the best way is to **obtain a newer copy** from the source you got it in the
+first place.
 
-If a newer version of Netdata is not available from the source you got it, we suggest to uninstall the version you have and follow the [installation](README.md) instructions for installing a fresh version of Netdata.
+If a newer version of Netdata is not available from the source you got it, we suggest to uninstall the version you have
+and follow the [installation](README.md) instructions for installing a fresh version of Netdata.
```

Comment:
They should never need to go through this, they should be going through the regular package manager based workflow for their system to update the package.